tests/integration/coordinator/plugins/clickhouse/test_plugin.py

```diff
@@ -43,15 +42,24 @@ def get_restore_steps_names() -> List[str]:
     return [step.__class__.__name__ for step in steps]
 
 
-@pytest.fixture(scope="module", name="restorable_cluster")
-async def fixture_restorable_cluster(ports: Ports, clickhouse_command: ClickHouseCommand) -> AsyncIterator[Path]:
+def _remove_frozen_tables_method(flag: bool) -> str:
+    return "SystemUnfreeze" if flag else "SnapshotClear"
+
+
+@pytest.fixture(scope="module", name="restorable_cluster", params=[True, False], ids=_remove_frozen_tables_method)
```

Comment:
Sorry, I missed it earlier:
It makes more sense to parametrize `use_system_unfreeze` on the restored cluster instead of the restorable cluster.
In the restorable cluster fixture, we never do a restore operation, we only do backup, so we don't exercise the unfreeze code.

Reply:
The restorable cluster takes the backup, so I set `use_system_unfreeze` in its config such that the flag is active during `RemoveFrozenTablesStep`. Am I missing something?
Did you rather mean to indirectly parametrize `fixture_restorable_cluster` using params in `fixture_restored_cluster` and build the test cases there?

Reply:
You didn't miss anything,  the test you wrote is completely fine.

I somehow mixed backup and restore while thinking about it :facepalm: